paper/paper.Rmd

```diff
@@ -50,20 +60,59 @@ knitr::opts_chunk$set(
   warning = FALSE
 )
 
+options(width = 60)
+
 library(report)
 set.seed(2016)
 ```
 
 # Summary
 
-The `{report}` package for the R programming language [@base2021] provides
+The `{report}` package is a powerful tool for researchers who use R to analyze data and create reports for publication. It bridges the gap between R's output and the formatted results to be included in a manuscript, allowing users to easily convert statistical models and data frames into textual reports that are suitable for publication. This paper provides an overview of this package, including its features, advantages, and how to use it. We also provide examples of how the package can be used to standardize and improve results reporting in R.
 
 # Statement of Need
 
-The `{report}` package is part of `{easystats}`, a collection of R packages designed to make statistical analysis easier (@Ben-Shachar2020, @Lüdecke2020parameters, @Lüdecke2020performance, @Lüdecke2021see, @Lüdecke2019, @Makowski2019, @Makowski2020, @Patil2022datawizard). 
+Reporting results is a crucial part of scientific research. Researchers need to ensure that their results are reported accurately and clearly, so that other researchers can understand and reproduce their work. However, the process of results reporting can be time-consuming and error-prone, particularly when it comes to formatting data for publication. The `{report}` package was developed to address this problem, providing researchers with a tool that can help standardize and improve reporting results from their workflows in R.
 
 # Features
 
+The package includes several features that make it a valuable tool for researchers. It can convert statistical models and data frames into textual reports, which can be easily customized and formatted to meet the requirements of a particular journal or publication. The package supports a wide range of output formats, including HTML, LaTeX, and Microsoft Word, via RMarkdown.
+
+Using this package in an analytic workflow has several advantages. First, it can help standardize results reporting across a research team or project, ensuring that everyone is using the same format and style. Second, the package can save researchers time by automating many of the tasks involved in reporting results, such as formatting tables and text. Third, the package can improve the quality of results reporting by providing users with a tool that is specifically designed for this purpose. Additionally, it prevents any copy-and-paste errors.
+
+The examples below illustrate the ease with which `{report}` can create detailed textual or tabular summaries for statistical objects:
+
+```{r}
```

Comment:
Wouldn't it be nice to have the `results='asis'` so they print as plain text?

Reply:
Maybe try adding a box? See [TEST.zip](https://github.com/easystats/report/files/10834634/TEST.zip)

Reply:
But this is beautiful! Will try it!

Reply:
Works! Looks great now. Updated paper with boxes: https://github.com/easystats/report/blob/joss-paper-draft/paper/paper.pdf